src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureManagementKeyValueAdapter.cs

```diff
@@ -77,10 +85,18 @@ public FeatureManagementKeyValueAdapter(FeatureFilterTracing featureFilterTracin
                             featureFlag.Conditions.RequirementType));
                     }
                 }
+
+                keyValues.Add(new KeyValuePair<string, string>($"{featureFlagPath}:{FeatureManagementConstants.Status}", FeatureManagementConstants.Conditional));
```

Comment:
Assuming we remove the "FF: true" shortcut, I prefer moving this to the top of this block since it's the most important "enabled" field.